package.json

```diff
@@ -17,8 +17,8 @@
     "jest": "jest --coverage test/jest/*.js",
     "predocs": "rm -rf ./jsdocs && mkdir jsdocs/ && ./utils/make-jsdoc-readme.js > ./jsdocs/jsdoc-temp.md",
     "docs": "./node_modules/.bin/jsdoc src/ -c ./docs/config.json -d ./jsdocs/ -P ./package.json -R ./jsdocs/jsdoc-temp.md -r",
-    "test:integration": "nyc --reporter=text --reporter=html mocha test/it/*.js --no-timeouts",
-    "test:unit": "nyc --reporter=text --reporter=html  mocha test/unit/*.js --no-timeouts",
+    "test:integration": "nyc --reporter=text --reporter=html mocha test/it/*.js --no-timeouts --reporter mocha-junit-reporter",
```

Comment:
IIRC - this outputs a `./test-results.xml` file with test output. If it does, can you make sure this file gets omitted in our `.gitignore`?